.github/workflows/integration-tests.yml

```diff
@@ -93,11 +93,8 @@ jobs:
       - run: git clone https://github.com/bjlittle/geovista.git --single-branch
       - name: Install PyVista
         run: pip install -ve . # pyvista
-      - run: sudo apt-get install libgeos-dev
       - name: Install GeoVista
-        run: pip install -ve . # geovista
-        working-directory: geovista
-      - run: pip install -r requirements/pypi-optional-test.txt
+        run: pip install -ve .[test] # geovista
```

Comment:
Seems to make sense (to me) to install all the PyPI packages for `geovista` at once ... or is there a good reason why this was previously performed in two steps ?